manifests/mod/proxy_connect.pp

```diff
@@ -2,17 +2,14 @@
 #   Installs `mod_proxy_connect`.
 # 
 # @param apache_version
-#   Used to verify that the Apache version you have requested is compatible with the module.
+#   Deprecated and unused. Remains here for compatibility.
 # 
 # @see https://httpd.apache.org/docs/current/mod/mod_proxy_connect.html for additional documentation.
 #
 class apache::mod::proxy_connect (
   $apache_version  = undef,
```

Comment:
I kept this variable in to not make it a breaking change. Silently ignoring it is safe, but it should be removed with the next major release.

Reply:
should we open a PR that drops it? I'm quite sure otherwise we will forget it.

Reply:
I thought about the same thing.